lix/packages/sdk/src/query-utilities/get-lowest-common-ancestor.ts

```diff
@@ -0,0 +1,54 @@
+import type { Change, LixReadonly } from "@lix-js/sdk";
+
+/**
+ * Finds the lowest common change of the source
+ * change in the source lix and the target lix.
+ *
+ * https://en.wikipedia.org/wiki/Lowest_common_ancestor
```

Comment:
if you need to add a wikipedia link to some math article, the naming is bad. i dont see any reason to add "lowest"

Reply:
Because you can have multiple common ancestors. Which one do you want? The lowest one in the tree. Hence, the name

Reply:
@samuelstroschein  no one would expect to get anything except the first common ancestor. this is how git and any other tool work.

Reply:
That doesn't seem true. Git does not use the first ancestor but the "best common ancestor" https://git-scm.com/docs/git-merge-base. And here we already have the problem. What common ancestor are you looking for? 

I'll keep the name lowest common ancestor because it describes the query. Thereby, we can add `findBestCommonAncestor` or other queries in the future. 

PS we should name it `find` instead of `get` to align with tree terminology.